system-tests/test/record_spec.js

```diff
@@ -883,6 +883,19 @@ describe('e2e record', () => {
         })
       })
 
+      it('errors and exits with 1 when posix exit codes are enabled', function () {
```

Comment:
whats the reason the posix tests are added here and not in the posix PR? Is there some type of integration between these two I am missing?

Reply:
oh I see its the differences between 112 vs 1 exit code